test/entities/swaps/v3/swapV3.integration.test.ts

```diff
@@ -221,7 +215,9 @@ describe('SwapV3', () => {
         snapshot = await client.snapshot();
     });
 
-    describe('query method should return correct updated', () => {
+    // TODO: double check if comparing query outputs against balanceDeltas isn't redundant with query tests
+    // if yes, we should be able to remove query tests (and avoid relying on a fixed blockNumber)
+    describe.skip('query method should return correct updated', () => {
```

Comment:
Highlighting this as it might be the one that sparks a bit of discussion.
Main question is: 
> do we need to test queryOutputs against hardcoded amounts?

I have a feeling that testing `queryOutput` against `balanceDeltas` from the actual transaction should be enough.
If we manage to remove these, we won't need to rely on a fixed blockNumber for these tests.

Reply:
Keeping in mind that after we launch there will be no new deployments and fixed block number will be permanent so it's probably not going to be that important soon. The main benefit I see is it potentially catches if changes break anything unexpectedly (e.g. results should always be the same). Its maybe not that important, especially if we're definitely checking that the query result is at least > 0 in `assertSwapExactIn/Out` but I'm not 100% sure that is the case and it would need to be confirmed first.